<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Body Mass Index Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="weight_slider">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QSlider" name="height_slider">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="gender_label_static">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLabel" name="gender_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="weight_label_static">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight (kg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="height_label_static">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Height (cm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="bmi_label_static">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Body mass index</string>
    </property>
   </widget>
   <widget class="QLabel" name="interpretation_label_static">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpretation</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="weight_box">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> kg</string>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="height_box">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> cm</string>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLabel" name="bmi_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="interpretation_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>weight_box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>weight_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>height_box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>height_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weight_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>weight_box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>height_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>height_box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
